.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Thank you for helping us improve the Azure IoT Edge!
```

Comment:
Shall we comment here that, for now, PR are accepted for v1?

Reply:
I added text, but made it general - we will accept pull requests for the V2 sample code as well.